app/components/SignupForm/SignupForm.js

```diff
@@ -88,29 +86,13 @@ class SignupForm extends BaseForm {
 
     if (this.props.signupSuccess !== nextProps.signupSuccess) {
       this.setState({
-        signupSuccess: nextProps.signupSuccess,
-        signupMessage: nextProps.isDuplicate
-          ? <li>
-              <p>
-                An account with this email domain already exists. Someone in your team may have already created an
-                account with the Marketplace
-              </p>
-            </li>
-          : <li>
-              <p>
-                The Digital Marketplace encountered an error trying to send your signup email. Please try again later or{' '}
-                <a href="/contact-us" target="_blank" rel="external">
-                  {' '}contact us{' '}
-                </a>{' '}
-                for assistance.
-              </p>
-            </li>
+        signupSuccess: nextProps.signupSuccess
```

Comment:
with the html removed from `signupMessage` we should be able to remove signupSuccess from component state and rely on `this.props.signupSuccess`